meilisearch/client.py

```diff
@@ -464,3 +468,67 @@ def wait_for_task(
             An error containing details about why Meilisearch can't process your request. Meilisearch error codes are described here: https://docs.meilisearch.com/errors/#meilisearch-errors
         """
         return wait_for_task(self.config, uid, timeout_in_ms, interval_in_ms)
+
+    def generate_tenant_token(
+        self,
+        search_rules: Dict[str, Any],
+        expires_at: Optional[int] = None,
```

Comment:
Yes, I should maybe say an Epoch timestamp like `1960621829`. At first, I use a `float` but I thought that an `int` is enough no? What do you think?

Reply:
Anyway, I will change it for a `datetime` so...

Reply:
Yes `datetime` would be good. How you handle it from there, either way is fine I think. It was only confusing before because Python’s `timestamp` returns a float.